src/content/headsup/index.css

```
@@ -1,12 +1,17 @@
 .r2c-repo-headsup {
   border: 1px solid #dfe2e5;
   border-radius: 3px;
-  margin-top: 10px;
 
   display: flex;
   flex-direction: column;
 }
 
+.checklist-headsup.detailed {
+  border-top-style: none;
+  border-top-left-radius: 0px;
```

No need for units when it's `0`
```suggestion
  border-top-left-radius: 0;
```